systemtest/src/test/java/io/strimzi/systemtest/KafkaClusterTest.java

```diff
@@ -304,4 +306,36 @@ public void testDeployKafkaOnPersistentStorage() {
         assertThat(configMap, valueOfCmEquals("KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", "5"));
         assertThat(configMap, valueOfCmEquals("KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", "5"));
     }
-}
+
+    @Test
+    @KafkaCluster(name = "my-cluster", kafkaNodes = 1, zkNodes = 2, config = {
+            @CmData(key = "zookeeper-healthcheck-delay", value = "30"),
+            @CmData(key = "zookeeper-healthcheck-timeout", value = "10"),
+            @CmData(key = "kafka-healthcheck-delay", value = "30"),
+            @CmData(key = "kafka-healthcheck-timeout", value = "10"),
+            @CmData(key = "KAFKA_DEFAULT_REPLICATION_FACTOR", value = "2"),
+            @CmData(key = "KAFKA_OFFSETS_TOPIC_REPLICATION_FACTOR", value = "5"),
+            @CmData(key = "KAFKA_TRANSACTION_STATE_LOG_REPLICATION_FACTOR", value = "5")
+    })
+    @OpenShiftOnly
```

Comment:
Can this test case be a cross-platform test? I don't see some big reason to use @OpenShiftOnly because we just do some general actions that can be available for both platforms. WDYT?